PICMI_Python/diagnostics.py

```diff
@@ -163,8 +163,8 @@ class PICMI_ParticleDiagnostic(_ClassWithInit) :
     period: integer
         Period of time steps that the diagnostic is performed
 
-    species: species instance or list of species instances
-        Species to write out.
+    species: species instance or list of species instances or None
```

Comment:
This might be a better ways of saying this
```
   species: species instance or list of species instances, optional
        Species to write out. If not specified, all species will be written out.
```